pages.es/common/qownnotes.md

```diff
@@ -0,0 +1,21 @@
+# qownnotes
+
+> Aplicación de toma de notas en formato Markdown.
+> Se integra opcionalmente con las aplicaciones de toma de notas de Nextcloud y ownCloud.
+> Más información: <https://www.qownnotes.org/getting-started/cli-parameters.html>.
+
+- Ejecuta en modo portable:
+
+`QOwnNotes --portable`
+
+- Vuelca la configuración y cualquier otra información sobre la aplicación y su entorno en formato GitHub Markdown:
+
+`QOwnNotes --dump-settings`
+
+- Especifica un contexto diferente para la configuración y los archivos internos:
+
+`QOwnNotes --session {{prueba}}`
+
+- Activa una acción de menú después de iniciar la aplicación:
+
+`QOwnNotes --action {{acciónMostrar_Lista_Quehacer}}`
```

Comment:
This page might be an exception to the rule of writing placeholders in snake case because the arguments seem to require the string `action` and some other stuff

> You can get the object names of the menu action from mainwindow.ui (opens new window). Just search for the English menu title. Note that these texts can change over time.

[Source](https://www.qownnotes.org/getting-started/cli-parameters.html#trigger-menu-actions-after-startup)

```suggestion
`QOwnNotes --action {{actionShow_Todo_List}}`
```